internal/driver/interactive_test.go

```diff
@@ -85,6 +85,19 @@ func TestShell(t *testing.T) {
 	if err := interactive(p, o); err != nil {
 		t.Error("invalid group value:", err)
 	}
+
+	// No value provided for the option
+	pprofVariables = testVariables(savedVariables)
```

Comment:
It would be nice to see test cases for a couple of other flags.

Reply:
Done.